### ISSUE TITLE
Zokrates JS proof different from CLI.

### ISSUE THREAD

**Issue by GeraldHost** [Zokrates/ZoKrates#1019]
## Description

Proof from CLI vs zokrates.js are different.

## Environment

- Compiler version:  0.7.3
- Operating system: Mac OSX
- Zokrates JS version:  1.0.35

## Steps to Reproduce

Zok source
```
import "hashes/keccak/256bit" as keccak

def swap_u64(u64 val) -> u64:
    val = ((val << 8) & 0xFF00FF00FF00FF00) | ((val >> 8) & 0x00FF00FF00FF00FF)
    val = ((val << 16) & 0xFFFF0000FFFF0000) | ((val >> 16) & 0x0000FFFF0000FFFF)
    return (val << 32) | (val >> 32)

def main() -> u64[4]:
    u64[12] inputs = [0, 0, 0, 1, 0, 0, 0, 2, 0, 0, 0, 3]
    
    // swap endianness
    for u32 i in 0..12 do
        inputs[i] = swap_u64(inputs[i])
    endfor

    // hash
    u64[4] digest = keccak(inputs)

    // swap endianness
    for u32 i in 0..4 do
        digest[i] = swap_u64(digest[i])
    endfor

    return digest
```

Javascript code
```js
const { initialize } = require("zokrates-js/node");

async function main() {
  const zok = await initialize();

  const source = '...';

  const artifacts = zok.compile(source);

  const { witness, output } = zok.computeWitness(artifacts, []);

  console.log(witness);

  console.log(output);

  const keypair = zok.setup(artifacts.program);

  const proof = zok.generateProof(artifacts.program, witness, keypair.pk);  

  console.log(proof);
}

if (require.main === module) {
  main();
}
```

The proof when generated from the cli is correct. But the proof from zokrates JS is all 0s. eg:

Bad proof
```
inputs: [
	'0x0000000000000000000000000000000000000000000000000000000000000000',
	'0x0000000000000000000000000000000000000000000000000000000000000000',
	'0x0000000000000000000000000000000000000000000000000000000000000000',
	'0x0000000000000000000000000000000000000000000000000000000000000000'
]
```

Good proof
```
"inputs": [
	"0x0000000000000000000000000000000000000000000000006e0c627900b24bd4",
	"0x00000000000000000000000000000000000000000000000032fe7b1f713f1b07",
	"0x00000000000000000000000000000000000000000000000044091a646a9fe4a6",
	"0x0000000000000000000000000000000000000000000000005a18dfed21f2949c"
]
```

**Comment by dark64** [comment]
Hey @GeraldHost, this is related to https://github.com/Zokrates/ZoKrates/pull/998. It is a bug with `u64` which keccak algorithm uses. This has been fixed and it's on `develop` branch, its gonna be available in the next release very soon.

**Comment by dark64** [comment]
Fixed in `>=0.7.7`
